packages/react-router-native/Link.js

```diff
@@ -41,8 +41,6 @@ class Link extends React.Component {
   }
 }
 
-const __DEV__ = true; // TODO
```

Comment:
We don't actually have a build for `react-router-native` yet, so we still need this.